tests/conftest.py

```diff
@@ -211,6 +211,16 @@ def jupyter_server():
 
     yield server_url
 
+    # Clean up any lingering kernel connections
+    try:
+        from mcp_jupyter.server import kernel
```

Comment:
Can we move this import to top of module here and elsewhere? Or does it need to be in the `try` block?